app/views/users/index.html.erb

```diff
@@ -13,11 +13,9 @@
 
 <% @users.each do |user| %>
   <tr>
-    <td><%= user.name %></td>
+    <td><%= link_to user.name, user_path(user) %></td>
     <td><%= user.email %></td>
-    <td><%= link_to 'Show', user_path(user) %></td>
-    <td><%= link_to 'Edit', edit_user_path(user) %></td>
```

Comment:
This, however, is actually a loss: now an admin needs 3 clicks to edit a user's profile, instead of one: Impersonate, click the name in the header, then Edit Profile.